CHANGELOG.md

```diff
@@ -38,10 +38,10 @@ const { DynamicThreadPool } = require('poolifier')
 For cluster and thread pools, you can now only send and receive serializable `JSON` data.  
 _This is not a limitation by poolifier but NodeJS._
 
-#### Public properties renaming
+### Public methods removed
```

Comment:
They are not methods? They always were properties

Reply:
Getters are methods, are they not?

Reply:
They are not!

Reply:
And even if, theses getters were added by me for backwards compatibility and were properties before in v1.2.1

Reply:
Oh and I see, we need a h4 headline here instead of h3
So please readd the missing `#`

Reply:
Thanks to catch @Shinigami92  done directly on master